e2e/run_test.go

```diff
@@ -232,3 +233,52 @@ func TestRunSpecialCharactersInPrefix(t *testing.T) {
 
 	assertLines(t, result.Stderr(), map[int]compareFunc{})
 }
+
+func TestRunDryRunAll(t *testing.T) {
+	t.Parallel()
+
+	bucket := s3BucketFromTestName(t)
+
+	s3client, s5cmd, cleanup := setup(t)
+	defer cleanup()
+
+	createBucket(t, s3client, bucket)
+
+	files := [...]string{"cpfile.txt", "mvfile.txt", "rmfile.txt"}
+	for _, f := range files {
+		putFile(t, s3client, bucket, f, "content")
+	}
+
+	filecontent := []string{
+		fmt.Sprintf("cp s3://%v/%s s3://%v/cp_%s", bucket, files[0], bucket, files[0]),
+		fmt.Sprintf("mv s3://%v/%s s3://%v/mv_%s", bucket, files[1], bucket, files[1]),
+		fmt.Sprintf("rm s3://%v/%s", bucket, files[2]),
+	}
+
+	file := fs.NewFile(t, "prefix", fs.WithContent(strings.Join(filecontent, "\n")))
+	defer file.Remove()
+
+	cmd := s5cmd("run", "--dry-run", file.Path())
```

Comment:
could you add variations of this test?

- try with `s5cmd run --dry-run-all`
- try with `s5cmd run --dry-run-all` and the payload contains `cp --dry-run foo bar`
- try with `s5cmd run --dry-run` and the payload contains `cp --dry-run foo bar`

Reply:
awesome 💯